tests/test_agile_sprints.py

```
@@ -0,0 +1,216 @@
+    def test_add_story(self):
+        s = AgileSprint(sprint_id="sp1", name="S1")
+        s.add_story(UserStory(story_id="s1", title="Login"))
+        assert s.story_count == 1
+        assert s.total_points == 1
+
+    def test_add_duplicate_raises(self):
+        s = AgileSprint(sprint_id="sp1", name="S1")
+        s.add_story(UserStory(story_id="s1", title="x"))
+        try:
+            s.add_story(UserStory(story_id="s1", title="y"))
+            assert False, "Expected ValueError"
```

Ruff (0.15.15)

[warning] Do not `assert False` (`python -O` removes these calls), raise `AssertionError()`

Replace `assert False`

(B011)